### ISSUE TITLE
Rewrite your code to use functions.

### ISSUE THREAD

**Issue by stephantul** [sanderhvn/Solving-P2#2]
You use one "piece of code" per file, which is good. 

Now try to join all pieces of code by turning them into functions. This allows you to remove the duplication you currently have in your `solution.py` file (which currently contains all the code you have written in your other files.)

**Comment by sanderhvn** [comment]
I am not sure what you mean. Should I turn each step into a function and bring all three together in `solution.py` ? Is the `solution.py` file not supposed to contain all code from the "subfiles"?

**Comment by stephantul** [comment]
> Should I turn each step into a function and bring all three together in solution.py ?
Yes. Try to take the exact code you wrote, and put them into a function, and simply `return` the values you used. You can then `import` these functions in `solution.py` and reuse the code. 

> Is the solution.py file not supposed to contain all code from the "subfiles"?
Not really. This is code duplication, and should be avoided. 

As an example, let's say you discover a bug in one of your subfiles. If you wrote every piece of code exactly once, in a function, and then reused this function, you will only have to fix this bug once. Now, you'd have to fix it 2 times, once in your subfile, and once in `solution.py`.

**Comment by sanderhvn** [comment]
fixed, i think!

**Comment by stephantul** [comment]
There's still some errors in your code:

your functions have the following signature:
`def evennums(lst):`

But you don't use the defined parameter `lst` in your function. So, the value of this parameter has no effect on the outcome of the function (i.e., no matter what value we set `lst` to, `evennums` will return the same value). 

I also can't run `solution.py`. The modules `step1`, `step2` etc. do not exist. Even if the import statements were correct there would still need to be code to actually call the functions you are importing.

something like this:
```
from xyz import my_function
from abc import my_other_function

result = my_function(some_input)
real_result = my_other_function(result)
```
